classes/item.rb

```diff
@@ -1,7 +1,7 @@
 require 'date'
 
 class Item
-  attr_reader :id, :archived, :genre, :author
+  attr_reader :id, :archived, :genre, :author, :label
   attr_accessor :publish_date
 
   def initialize(publish_date, archived)
```

Comment:
Be sure to add the `@label` instance variable to the initialize method in the `Item` class.
You can set it to `nil` since we are going  to set it later.

Reply:
👍